hera/nfs/shared.py

```diff
@@ -0,0 +1,8 @@
+from hera.shared import global_config
```

Comment:
I'd add 
```python
from hera.workflows import *
from hera.workflows import models, models as m
```

This way all the other modules can import `Workflow, Parameter, etc` from this module, and they get the global setup with it, rather than import from hera + `import shared` in order for the global config to actually run. 

See this as an example of [setting up a "common/personal module"](https://github.com/flaviuvadan/kubecon_na_23_llama2_finetune/blob/main/src/talk/workflows.py) and [usage of it](https://github.com/flaviuvadan/kubecon_na_23_llama2_finetune/blob/main/src/talk/train.py#L23)

Reply:
Neat, done that.